lib/imagec/imagec.go

```diff
@@ -138,7 +138,7 @@ var (
 
 const (
 	// DefaultDockerURL holds the URL of Docker registry
-	DefaultDockerURL = "registry.hub.docker.com"
+	DefaultDockerURL = "registry-1.docker.io"
```

Comment:
This honestly seems like something that should be configurable so we can change the default registry. Doing so would also allow adjustment of this url if/when we find other impacts. Not needed for this PR but interesting in knowing viability.

Reply:
registry-1.docker.io is hard coded in docker code.  It's not a configurable option, but yes, doing so would give imageC some flexibility.